manifests/package/debian.pp

```diff
@@ -37,21 +37,21 @@
         apt::source { 'nginx':
           location => "https://nginx.org/packages/${distro}",
           repos    => 'nginx',
-          key      => '573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62',
+          key      => {'id' => '573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62'},
```

Comment:
Here too... @bastelfreak is this Ok, since this is one that was already merged, just kind of confusing commit message.... Or should I revert?